recipes/dirty_cat/0001-remove-benchmark.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.py	2021-12-19 11:54:50.000000000 -0500
++++ setup.py	2022-01-10 17:34:31.594365500 -0500
+@@ -36,7 +36,7 @@
+               'Topic :: Software Development :: Libraries',
+           ],
+           platforms='any',
+-          packages=find_packages(),
++          packages=find_packages(exclude=["benchmark"]),
```

Comment:
"benchmarks"

Reply:
whoops. Yeah also I guess I need to take out the `import benchmarks test`